docs/lang/articles/math/math_module.md

```diff
@@ -49,34 +47,33 @@ def test():
 
 :::note
 
-Difference between Taichi's math module and Python's built-in math module:
+Taichi's math module has a large overlap with Python's built-in math module. But you should follow a few extra rules when using Taichi's math module:
 
-- Functions in Taichi's math module must be called from within the Taichi scope.
-- Functions in Taichi's math module also accept vectors/matrices as arguments.
-- The precision of a function in Taichi's math module depends on the value of `default_fp` and backend specified in the `ti.init()` method call.
++ Functions in Taichi's math module *must* be called from within the Taichi scope.
```

Comment:
Active voice. Use active voice as much as possible. 

```suggestion
- You must call the functions provided by Taichi's math moduled from within the Taichi scope.
```
use "-" for bullet. Please update the entire document.